_posts/2018-02-26-success-with-serverless.md

```diff
@@ -28,15 +28,17 @@ While we did make this solution work, it was not ideal. Geoprocessing services,
 In the end, we decided that there was no need to incur all of the heavy overhead of ArcGIS Server for a simple proxy service.
 
 ### Serverless to the Rescue
+This is where serverless computing came in. 
+
 Serverless computing centers around a simple concept: abstracting away all of the pain that comes from managing systems and allowing developers to focus on building software. With serverless computing, you simply write the code and let the experts ([Google](https://cloud.google.com/functions/), [Amazon](https://aws.amazon.com/lambda/), [Microsoft](https://azure.microsoft.com/en-us/services/functions/), and others) take care of all of the headaches associated with deploying and hosting. And even better, you only pay when your service is actually invoked. Consequently, the cost ends up being pennies on the dollar compared to standing up a traditional server. In fact, AGRC, to date, has not crossed the threshold of the free tier.
 
-Many of the major vendors provide command-line utilities to help you get up and running quickly. We decided to give the provider-agnostic project, [serverless](https://serverless.com) a try. [Getting started](https://serverless.com/framework/docs/providers/google/guide/quick-start/) was fairly simple:
+Many of the major vendors provide command-line utilities to help you get up and running quickly. We decided to give the provider-agnostic project, [Serverless](https://serverless.com), a try. [Getting started](https://serverless.com/framework/docs/providers/google/guide/quick-start/) was fairly simple:
```

Comment:
_We decided to give the provider-agnostic project, [Serverless](https://serverless.com), a try.:_ "Serverless" in this sentence is referring to the proper name of a product, right? If so, I think we should cap it.

Reply:
correct. their logo is lowercase but they use proper case through out their website.

Reply:
Ok that's what I thought! Good good.